projects/wallet-template/README.md

```diff
@@ -1,2 +1,105 @@
-## Substrate Connect Wallet Template
+# Substrate Connect - Light Client Wallet Template
 
+## Overview
+
+Welcome to Substrate Connect - the ultimate light client wallet template! Substrate
+Connect leverages the power of the smoldot,
+a light client that implements the JSON RPC Spec to provide a highly resilient, 
+efficient, and user-friendly wallet experience. This template is designed to help 
+developers quickly build and deploy a decentralized wallet that interacts with 
+any blockchain in the Substrate ecosystem using a light client.
+
+What's more, when you use Substrate Connect as a browser extension, your dapps 
+can instantly connect to popular Substrate-based chains without the hassle of 
+waiting for synchronization. This is possible because Substrate Connect keeps 
+track of the chains in the background, even when the user isn't actively 
+connected to any dapps. At the same time, the dApp will show up as "connected" 
+within the extension, indicating that it is leveraging the extension's bootnodes 
+and specifications.
+
+## Getting Started
+
+### Prerequisites
+
+- **Corepack** - [Zero runtime dependency manager](https://github.com/nodejs/corepack)
+
+### Installation
+
+Clone the repository and install dependencies:
+
+```bash
+corepack pnpm i
+corepack pnpm turbo build
+```
+
+### Running the Wallet
+
+Start the development server:
+
+```bash
+corepack pnpm dev
+```
+
+In another terminal:
+
+```bash
+corepack pnpm start
+```
+
+This will open the extension in a browser window.
+
+## Usage
+
+### As a Wallet
+
+1. Launch the extension and create a password
+
+![step 1](./assets/img/step1.png)
+
+2. Click "Add" and create your first crypto key
+
+![step 2](./assets/img/step2.png)
+
+3. You now have access to your accounts. Each account is linked to a specific
+derivation path for each consensus system.
+
+![step3](./assets/img/step3.png)
+
+### As a Dapp
+
+As a dapp, you will want to use the [@substrate/discovery](../../packages/discovery/README.md)
+protocol to discover the extension. An example implementation can be found in the
+[light client dapp](../../examples/light-client-dapp/) folder.
+
+### As an extension
+
+You will want to utilize the [@substrate/light-client-extension-helpers](../../packages/light-client-extension-helpers/README.md) 
```

Comment:
ahh its because i split the PR up, its there on main now: https://github.com/paritytech/substrate-connect/tree/main/packages/light-client-extension-helpers